compass/landice/tests/ismip6_run_ais/__init__.py

```diff
@@ -0,0 +1,20 @@
+from compass.testgroup import TestGroup
+from compass.landice.tests.ismip6_run_ais.projection import Projection
+
+
+class Ismip6RunAIS(TestGroup):
```

Comment:
Should ideally be:
```suggestion
class Ismip6RunAis(TestGroup):
```
(CamelCase rather than all caps).